webapp/src/main/java/com/box/l10n/mojito/rest/asset/AssetWS.java

```diff
@@ -238,6 +243,24 @@ public XliffExportBody xliffExport(@PathVariable("assetId") long assetId, @Reque
         return new XliffExportBody(normalizedContent);
     }
 
+    /**
+     * Exports all the translations (used and unused) of an {@link Asset} into
+     * XLIFF asynchronously
+     *
+     * @param assetId {@link Asset#id}
+     * @param bcp47tag bcp47 tag of translations to be exported
+     * @return a {@link PollableTask} that generates XLIFF asynchronously in a
+     * {@link XliffExportBody}
+     */
+    @RequestMapping(method = RequestMethod.POST, value = "/api/assets/{assetId}/xliffExport")
+    public XliffExportBody xliffExportAsync(@PathVariable("assetId") long assetId,
+            @RequestParam("bcp47tag") String bcp47tag,
+            @RequestBody XliffExportBody xliffExportBody) {
+        PollableFuture pollableFuture = tmService.exportAssetAsXLIFFAsync(assetId, bcp47tag, PollableTask.INJECT_CURRENT_TASK);
```

Comment:
Similarly to `AssetWS#importSourceAsset` that WS should create the initial `TMXliff` entity and return it along with the pollable id. 

Then the GET WS would take the id of the `TMxliff` entity and return it.